extension/popup.html

```diff
@@ -32,23 +32,23 @@ <h1 class="settings-hl">Settings</h1>
             <button title="" class="data-collection fx-relay-toggle"></button>
           </li>
           <li class="setting flex-col">
+            <a href="https://www.mozilla.org/privacy/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup&utm_campaign=beta" class="setting-label close-popup-after-click" data-event-action="click" data-event-label="panel-privacy-link">Privacy</a>
```

Comment:
Should we match FPN? (Link "Terms of Service" to https://www.mozilla.org/en-US/about/legal/terms/firefox-relay/)
<img width="350" alt="image" src="https://user-images.githubusercontent.com/71928/90047914-4f316b80-dc98-11ea-92b6-f2365fc5a256.png">

Reply:
Yep - I just needed this URL (wasn't sure if/where it existed)/